JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.html

```diff
@@ -0,0 +1,201 @@
+<div>
```

Comment:
It looks like a redundant div container
Can we remove it and add CSS style :host { display: block; } to the app.component.css?

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/f39c14a692bbcd389fb88cb25511fb255f402ca5